workflows/sars-cov-2-variant-calling/sars-cov-2-pe-illumina-artic-ivar-analysis/pe-wgs-ivar-analysis.ga

```diff
@@ -0,0 +1,1142 @@
+            "tool_version": "4.5covid19",
+            "type": "tool",
+            "uuid": "a8b63b67-2795-4e21-8614-f7d24cec2224",
+            "workflow_outputs": [
+                {
+                    "label": "snpeff_annotated_vcf",
+                    "output_name": "snpeff_output",
+                    "uuid": "7f70af63-b3d1-437c-8ba4-4fedb1242431"
+                }
+            ]
+        },
+        "18": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sed_tool/1.1.1",
+            "errors": null,
+            "id": 18,
+            "input_connections": {
+                "infile": {
+                    "id": 15,
+                    "output_name": "consensus"
+                }
+            },
+            "inputs": [],
+            "label": "Consensus genome (masked for depth)",
+            "name": "Text transformation",
+            "outputs": [
+                {
+                    "name": "output",
+                    "type": "input"
+                }
+            ],
+            "position": {
+                "bottom": 1095.4666748046875,
+                "height": 134,
+                "left": 659.7666625976562,
+                "right": 859.7666625976562,
+                "top": 961.4666748046875,
+                "width": 200,
+                "x": 659.7666625976562,
+                "y": 961.4666748046875
+            },
+            "post_job_actions": {
+                "RenameDatasetActionoutput": {
+                    "action_arguments": {
+                        "newname": "Consensus genome"
+                    },
+                    "action_type": "RenameDatasetAction",
+                    "output_name": "output"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sed_tool/1.1.1",
+            "tool_shed_repository": {
+                "changeset_revision": "fb4ff3c42cd3",
+                "name": "text_processing",
+                "owner": "bgruening",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"adv_opts\": {\"adv_opts_selector\": \"basic\", \"__current_case__\": 0}, \"code\": \"/^>/s/Consensus_(.*)_threshold_.*/\\\\1/\", \"infile\": {\"__class__\": \"ConnectedValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "1.1.1",
+            "type": "tool",
+            "uuid": "877a609d-d2fa-430d-8b78-2b484af52bda",
+            "workflow_outputs": [
+                {
+                    "label": "ivar_consensus_genome",
+                    "output_name": "output",
+                    "uuid": "bf14144c-7147-44f1-a7b4-ee778ed87b11"
+                }
+            ]
+        },
+        "19": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cat/0.1.1",
```

Comment:
Just out of interest: @mvdbeek is this tool or "Collapse collection" better suited for concatenating the elements of a collection?

Reply:
tp_cat is simpler and has fewer options. It's a thin wrapper around cat, while `Collapse collection` has options to add the element_identifier and to keep one header line. Collapse collection` would probably reach a limit in the length of the generated command line for very big collections (https://github.com/bgruening/galaxytools/pull/1135) ... which could be fixed of course.

So for this use tp_cat is probably better.